deploy/06_rebased_staked_celo.ts

```diff
@@ -4,6 +4,7 @@ import { DeployFunction } from "hardhat-deploy/types";
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const stakedCelo = await hre.deployments.get("StakedCelo");
   const account = await hre.deployments.get("Account");
+  const multisig = await hre.deployments.get("MultiSig");
 
   const { deploy } = hre.deployments;
 
```

Comment:
remove `owner` from the namedAccounts. I dont think it's needed since we are using multisig as owner. I think this is left over from testing.